tests/nested/manual/core20-fde-dbx/task.yaml

```diff
@@ -12,10 +12,31 @@ environment:
     NESTED_UBUNTU_SEED_SIZE: 1500M
 
 prepare: |
-    # TODO:FDEM: copy nested vars file to "$NESTED_ASSETS_DIR/OVMF_VARS.snakeoil.fd"
-    #
+    tests.pkgs install efitools
+
     tests.nested build-image core
-    tests.nested create-vm core    
+    tests.nested create-vm core
+    remote.exec sudo snap install --devmode test-snapd-efitools
+    remote.exec sudo test-snapd-efitools.tool efi-readvar -v dbx 2>&1 | tee efi-dbx-state-initial.log
+    # count how many entries we have in DBX initially, this should be identical
+    # across UC20-UC24+ systems since we are using the same OVMF variables blob,
+    # but on UC20 2 additional entries get added at runtime
```

Comment:
In theory, secureboot-db is also on 22 and 24. Not sure why it would behave differently.